src/main/antlr4/soql/SOQL.g4

```diff
@@ -61,6 +61,11 @@ keyword             : GROUP
                     | CATEGORY
                     ;
 
+selectKey           : DATA
+                    | SCOPE
+                    | END
```

Comment:
On what basis are we selecting the keywords list here?

Reply:
These are the keywords that we have encountered from customers' usage. There is no specific list, we can add more if you can suggest.

Reply:
hm it seems like this is not a comprehensive list; we may hit similar issues in the future for other keywords.
Can we accept any identifier or keyword in the select clause?

Reply:
No, only the ones defined in the `SOQL.g4` file.

We went through all the standard objects [here](https://developer.salesforce.com/docs/atlas.en-us.234.0.object_reference.meta/object_reference/sforce_api_objects_list.htm) and found 1 new keyword as a field name that wasn't covered in the code.

Offset in [CommSubscriptionTiming](https://developer.salesforce.com/docs/atlas.en-us.234.0.object_reference.meta/object_reference/sforce_api_objects_commsubscriptiontiming.htm)

Added to the selectKey c6ac3065a0082add66d16d798a24ecd38d2272a7